apps/mobile-wallet/src/features/modals/BottomModal.tsx

```diff
@@ -63,10 +67,19 @@ export interface BottomModalProps {
 
 const AnimatedPressable = Animated.createAnimatedComponent(Pressable)
 
-const BottomModal = ({ Content, onClose, title, maximisedContent, customMinHeight, noPadding }: BottomModalProps) => {
+const BottomModal = ({
+  id,
+  Content,
+  onClose,
+  title,
+  maximisedContent,
+  customMinHeight,
+  noPadding
+}: BottomModalProps) => {
   const insets = useSafeAreaInsets()
-
+  const dispatch = useAppDispatch()
   const [dimensions, setDimensions] = useState(Dimensions.get('window'))
+  const modalEntity = useAppSelector((s) => selectModal(s, id))
```

Comment:
nit: We pretty much everywhere have named these selectors with a `ById` suffix.

Reply:
I see the 👍 but I see that it's not yet updated. Are you planning to? Just checking as I go through the diff again. Thanks!

Reply:
Done